src/oic/oic/provider.py

```diff
@@ -499,9 +499,8 @@ def pick_auth(self, areq, comparision_type=""):
             else:  # same as any
                 return self.authn_broker[0]
         except KeyError as exc:
-            logger.debug(
-                "An error occured while picking the authN broker: %s" % str(
-                    exc))
+            msg = "An error occurred while picking the authN broker: {}"
+            logger.debug(msg.format(str(exc)))
```

Comment:
Ah! I wasn't aware. Is [this still an issue?](https://www.reddit.com/r/Python/comments/387oog/when_will_logging_use_newstyle_string_formatting/). Should I use `%s` instead of `{}`?